ksql-engine/src/main/java/io/confluent/ksql/QueryEngine.java

```diff
@@ -159,41 +158,7 @@ private PlanNode buildQueryLogicalPlan(
     return logicalPlan;
   }
 
-  List<QueryMetadata> buildPhysicalPlans(
-      final List<LogicalPlanNode> logicalPlans,
-      final List<PreparedStatement> statementList,
-      final KsqlConfig ksqlConfig,
-      final Map<String, Object> overriddenProperties,
-      final KafkaClientSupplier clientSupplier,
-      final MetaStore metaStore,
-      final boolean updateMetastore
-  ) {
-
-    final List<QueryMetadata> physicalPlans = new ArrayList<>();
-
-    for (int i = 0; i < logicalPlans.size(); i++) {
-
-      final LogicalPlanNode node = logicalPlans.get(i);
-      if (node.getNode() == null) {
-        final Statement statement = statementList.get(i).getStatement();
-        if (!(statement instanceof DdlStatement)) {
-          throw new KsqlException("expecting a statement implementing DDLStatement but got: "
-                                  + statement.getClass());
-        }
-        handleDdlStatement(node.getStatementText(), (DdlStatement) statement, overriddenProperties);
-      } else {
-        buildQueryPhysicalPlan(
-            physicalPlans, node, ksqlConfig,
-            overriddenProperties, clientSupplier, metaStore, updateMetastore
-        );
-      }
-
-    }
-    return physicalPlans;
-  }
-
-  private void buildQueryPhysicalPlan(
-      final List<QueryMetadata> physicalPlans,
+  QueryMetadata buildPhysicalPlans(
```

Comment:
nit: this isn't returning a collection anymore, so rename to `buildPhysicalPlan`

Reply:
done